code/index.html

```diff
@@ -3,28 +3,119 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" >
+
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <header>
+      <div class="header-textbox">
+        <h1>Wildlife camp</h1>
+        <h3 class="slogan">Learn how to survive in the wild</h3>
+      </div>
+
+    </header>
+
+    <section class="navbar">
+      <button class="hamburger-nav" id="burger">
+        <div class="burger1"></div>
+        <div class="burger2"></div>
+        <div class="burger3"></div>
+      </button>
+    </section>
+
+    <section class= "main">
+      <section class="sidebar">
+        <div class="text-overlay toggle-display">
+          <video class="sidebar-video" width="350" height="350" muted loop>
```

Comment:
Try to add the styling in the css instead. You can target the video with `video {}`